build/templates/grpc_session_options.py.mako

```diff
@@ -55,23 +58,22 @@ class GrpcSessionOptions(object):
     ):
         r'''Collection of options that specifies session behaviors related to gRPC.
 
-        Creates and returns an object you can pass to a session constructor.
+        Creates and returns an object you can pass to a Session constructor.
 
         Args:
             grpc_channel (grpc.Channel): Specifies the channel to the NI gRPC Device Server.
 
-            session_name (str): Specifies the **session name** to be used in the NI gRPC Device
-                Server. This is different from the resource name parameter many APIs take as a
-                separate parameter. Specifying a name makes it easy to share sessions across
-                multiple gRPC clients. You can use an empty string if you want to always initialize
-                a new session on the server and then close it when the Python Session instance on
-                the client goes out of scope.
+            session_name (str): User-specified name that identifies the driver session in the NI gRPC Device
+                Server. This is different from the resource name parameter many APIs take as a separate
+                parameter. Specifying a name makes it easy to share sessions across multiple gRPC clients.
+                You can use an empty string if you want to always initialize a new session on the server.
+                To attach to an existing session, you must specify the session name it was initialized with.
```

Comment:
>To attach to an existing session, you must specify the session name it was initialized with.
Should this be "session name" or "resource name"?

Reply:
Discussed in chat; these are different names. No change.

Reply:
It was explained to me. The session name is different from the resource name as mentioned in the documentation above. The shared session use case is for a session already opened with GrpcSessionOptions. That is where the session name is specified for an already opened session is specified.